ui/packages/consul-ui/app/templates/dc/routing-config.hbs

```diff
@@ -0,0 +1,22 @@
+{{page-title @model.slug}}
+
+<AppView>
+  <BlockSlot @name="breadcrumbs">
+    <ol>
+        <li><a data-test-back href={{href-to 'dc.services'}}>All Services</a></li>
```

Comment:
I thought about moving this to translations but I thought it would be best to have all breadcrumbs moved to translations in a separate PR.